examples/chip-tool/commands/pairing/PairingCommand.h

```diff
@@ -58,6 +59,9 @@ class PairingCommand : public CHIPCommand,
         mFilterType(filterType), mRemoteAddr{ IPAddress::Any, chip::Inet::InterfaceId::Null() }
     {
         AddArgument("node-id", 0, UINT64_MAX, &mNodeId);
+        AddArgument("bypass-attestation-verifier", 0, 1, &mBypassAttestationVerifier,
+                    "Bypass the attestation verifier. If not provided or 0 (\"false\"), the attestation verifier is not bypassed."
```

Comment:
```suggestion
                    "Bypass the attestation verifier. If not provided or false, the attestation verifier is not bypassed."
```
In that you can and should do `--bypass-attestation-verifier false`